frontend/coprs_frontend/coprs/views/apiv3_ns/apiv3_project_chroots.py

```diff
@@ -75,35 +75,37 @@ def rename_fields(input_dict):
 
 @apiv3_project_chroots_ns.route("/")
 class ProjectChroot(Resource):
-    parser = project_chroot_parser()
-
-    @apiv3_project_chroots_ns.expect(parser)
+    @apiv3_project_chroots_ns.expect(project_chroot_get_input_model)
     @apiv3_project_chroots_ns.marshal_with(project_chroot_model)
     def get(self):
         """
         Get a project chroot
         Get settings for a single project chroot.
         """
-        args = self.parser.parse_args()
-        copr = get_copr(args.ownername, args.projectname)
-        chroot = ComplexLogic.get_copr_chroot(copr, args.chrootname)
+        # pylint: disable-next=unbalanced-tuple-unpacking
+        ownername, projectname, chrootname = request_multiple_args(
```

Comment:
the `request_multiple_args` returns only arg values that are specified as parameters and its guaranteed that the number of parameters == len(returned list). But pylint can't know... This approach is not ideal, I agree...

I can return a dict[str, Any], where str will be param name passed to function and Any will be its value (and on top of it it can be dict with dot access but I don't like these much). It would be accessible e.g. as d["ownername"] (or d.ownername)